### PR TITLE
Added new DA eligible case state

### PR DESCRIPTION
**Pull request by johnoneill46**
Added new "DARequested" state to list of valid DA case states

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -153,6 +153,7 @@ ccd:
     - 'serviceCentre'
   validDaStates:
     - 'AwaitingDecreeAbsolute'
+    - 'DARequested'
```
Comment (hosainnet) [inline config/default.yml]: Do we have the right content to show in this scenario?

Comment (hosainnet) [reply config/default.yml]: Also BTW we should have a tech debit to move this away from config - it's core the the app logic and won't be overriden per env so it should be in code and tested

Comment (johnoneill46) [reply config/default.yml]: Yep we have a PR waiting to be QA'd & merged for this case state:
https://github.com/hmcts/div-decree-absolute-frontend/pull/21/files

**Comment by vianvi** [comment]
https://tools.hmcts.net/jira/browse/DIV-5081
based on ticket this is right state and merging this now

**Pull request merged**